source/microbit/help.c

```diff
@@ -138,6 +138,7 @@ STATIC const mp_doc_t help_table_instances[] = {
     {&microbit_i2c_obj, "Communicate with one or more named devices connected to micro:bit. Each named\ndevice has an 'address', communicates using I2C, and connects to the I/O pins.\n"},
     {&microbit_i2c_read_obj, "Use read(address, n) to read 'n' bytes from the device with this address.\n"},
     {&microbit_i2c_write_obj, "Use write(address, buffer) to write to the 'buffer' of the device at this 'address'.\n"},
+    {&microbit_i2c_init_obj, "Set the bus frequency and pins to use.\n"},
```

Comment:
Sorry, I just realised we should follow existing messages.  Eg: "Use init(freq, sda, scl) to set the bus 'freq' and pins to use.\n".